api/cmd/main.go

```diff
@@ -29,18 +47,28 @@ func main() {
 		os.Getenv("POSTGRES_PORT"),
 		os.Getenv("POSTGRES_DB"),
 	)
-	db, err := sql.Open("postgres", con)
+	db, err := sqlx.Open("postgres", con)
 	if err != nil {
 		log.Fatal(err)
 	}
 	h := Handler{
-		database: db,
+		feeds:     vex.NewFeedService(db),
+		users:     vex.NewUserService(db),
+		jwtSecret: []byte(os.Getenv("JWT_SECRET")),
 	}
 
 	e := echo.New()
+	e.Validator = &Validator{validator: validator.New()}
 	e.Use(middleware.Logger())
+
+	r := e.Group("")
```

Comment:
useful ca?

Reply:
yes, used for restricted routes (based on jwt). the ""is the prefix but we don't want one.